### DISCUSSION TITLE
Could you please add more sponsoring options (e.g. PayPal) to your profile or your project?

### DISCUSSION THREAD

**Discussion by DwainZwerg** [ImranR98/Obtainium#1362] [Ideas]
Hello @ImranR98 ,
for instance in European countries (especially Germany, Austria (not so much the Scandinavian countries)), where the alternative EC card is used, it is still not common to have (real) credit cards (because the bank fees are usually much more expensive than those of an EC account). If one wants to transfer money to other countries outside the EU, one therefore usually uses PayPal. Could you imagine adding a PayPal account as another supporter option?
Thanks

**Comment by ImranR98** [comment]
https://github.com/ImranR98/Obtainium/issues/1369
